src/formats/onenote.ts

```diff
@@ -674,6 +674,12 @@ export class OneNoteImporter extends FormatImporter {
 		const videos: HTMLIFrameElement[] = pageElement.findAll('iframe') as HTMLIFrameElement[];
 
 		for (const object of objects) {
+			// Objects may contain child nodes which would be lost when the object is replaced by markdown.
+			// To preserve these, move any child items to be siblings of the object
+			while (object.firstChild) {
+				object.parentNode?.insertBefore(object.firstChild, object.nextSibling);
```

Comment:
Could `object.nextSibling` be null? Maybe this should be `object.after(object.firstChild)` instead? ([after](https://developer.mozilla.org/en-US/docs/Web/API/Element/after))

Reply:
Yes, it may be null, in which case it gets added to the end of the list

(https://developer.mozilla.org/en-US/docs/Web/API/Node/insertBefore#referencenode)

Reply:
Ah perfect, thank you.